src/Vim.xml

```diff
@@ -29,14 +29,14 @@
             </item>
                <item>
                   <bullet>2)</bullet>
-              The modified Vim must be distributed in one of the following five ways:
+              The modified <alt match=".+" name="softwareName">Vim</alt> must be distributed in one of the following five ways:
               <list>
                      <item>
                         <bullet>a)</bullet>
-                  If you make changes to Vim yourself, you must clearly describe in the distribution how to contact
-                    you. When the maintainer asks you (in any way) for a copy of the modified Vim you distributed,
+                  If you make changes to <alt match=".+" name="softwareName">Vim</alt> yourself, you must clearly describe in the distribution how to contact
+                    you. When the maintainer asks you (in any way) for a copy of the modified <alt match=".+" name="softwareName">Vim</alt> you distributed,
                     you must make your changes, including source code, available to the maintainer without fee.
-                    The maintainer reserves the right to include your changes in the official version of Vim. What
+                    The maintainer reserves the right to include your changes in the official version of <alt match=".+" name="softwareName">Vim</alt>. What
```

Comment:
I don't understand how making Vim replaceable here is compatible with hardcoding the maintainer of Vim's name in the following sentence.

Reply:
All "maintainer" in the license text refers to "Vim Maintainer" according to Bram [[1][]]

Which means if someone distributed a modified software with Vim License and not open source.
Bram as Vim maintainer should be the one to ask for the source.
But this case is very rare. Because Vim script/plugin are not easy to distribute without open source.

[1]: https://groups.google.com/d/msg/vim_dev/DlTVMew1ZSo/h1C0MTBYCgAJ

Reply:
Ok, that's odd but I understand, thanks for explaining. Reading with that in mind, I have at least one other comment elsewhere.

Reply:
Actually, in the same place. I still don't understand how replacing the Vim here makes sense given that maintainer always refers to the Vim maintainer. If person X releases software Y under the Vim License, isn't X and not the Vim maintainer in a position to make changes to the official version of Y?

I'd be curious to see an example of how this is done in the wild, I might be missing something obvious.

Reply:
The license text is old. I believe the first version is not consider other software to use it. So the word is not perfectly match it.

In this case. If there is a modified software Y called Y'. And the source code of Y' is not distributed. Vim maintainer should be the one to request it. And decide to give codes to X. Explanation by Bram[[2][]]:

> Note that "Vim maintainer" is different, that is not the product name
and should remain as-is.  In the bigger context this part is a bit
confusing, but we can't change it now.  The basic idea is that if
someone wants to make changes to [product name] but not publish the
changed code, the maintainer will have the right to decide about that.
This if you publish a plugin with the Vim license, I get to decide what
happens when someone makes a modified version and doesn't want to
publish it.  In practice this never happened, thus it's a precaution,
while keeping the door open for a commercial spin-off. 

Two points:

* Vim Script/Plugin is always distributed with source codes. This situation is rare to Vim scripts/Plugin.
* This never happened before. So there is no real case in the wild now.

[2]:https://groups.google.com/d/msg/vim_dev/DlTVMew1ZSo/QddqQZsMCgAJ

Reply:
I understand the part about only the Vim maintainer having the right to demand the source. That's not what I'm asking about here, rather:

> The maintainer reserves the right to include your changes in the official version of <alt match=".+" name="softwareName">Vim</alt>.

It seems like this instance of Vim ought not be replaceable, i.e., it should not be wrapped with `alt`. How is the Vim maintainer supposed to include your changes in the official version of software they don't maintain, ie non-Vim software? Make a PR/send a patch, I guess? OK, maybe that makes sense, even odder though in my book. Just thinking though it though, thanks for patiently explaining.

...

My question about 'in the wild' was not about whether Bram had made source code requests, but whether the Vim license had been used by other projects with Vim replaced: would this PR result in a SPDX template powered matcher running against the universe match any files that would not have been matched without this PR, or would it just expend more energy? :smile:

Reply:
> The maintainer reserves the right to include your changes in the official version of Vim.

I didn't confirm this part. But we can ask it at vim_dev. I just think for this case Bram don't need to really decide for the non-Vim software. He only decide should these changes open or close. Not 100% match the license text but ok for me.

In the wild, I know [tagbar][] uses modified version of Vim License. But Tagbar License(the modified version) replaced everything. Which might be what you are expect to see(my first version also replaces all these information).

I have 1 or 2 project use software name replaced Vim License text. For other projects using Vim License. Most of they didn't change the License text. But I don't think it's correct way to use Vim License. So my next step is to help them update their license text.

PS: I believe only several people know about this "Vim Maintainer" fact before I start working on this.

[tagbar]:https://github.com/majutsushi/tagbar/blob/master/LICENSE